### PR TITLE
Exercise 2 - Create a simple madlib form

### PR DESCRIPTION
**Pull request by ccjjmartin**
URL for the madlib form is:

/madlib

### CODE REVIEW

File: drupal_form_api.module [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+      3 => 4,
+      4 => 5,
+      5 => 6,
+      6 => 7,
+      7 => 8,
+      8 => 9,
+      9 => 10,
+    ),
+    '#default_value' => variable_get('drupal_form_api_number_select', ''),
+    //'#description' => t('Number'),
```
Comment (fluxsauce) [inline drupal_form_api.module]: Please remove commented out code.

**Comment by fluxsauce** [comment]
@ccjjmartin please merge into this branch and resolve conflicts.

**Comment by ccjjmartin** [comment]
Working on it now

File: drupal_form_api.module [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+  );
+  
+  $form['drupal_form_api_plural_noun_3_textfield'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Plural Noun Three'),
+    '#default_value' => variable_get('drupal_form_api_plural_noun_3_textfield', ''),
+    //'#description' => t('Plural Noun Three'),
```
Comment (fluxsauce) [inline drupal_form_api.module]: Remove commented out code.

File: drupal_form_api.module [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+    '#size' => 60,
+    '#maxlength' => 128,
+    '#required' => FALSE,
+  );
+  
+  $form['drupal_form_api_part_of_body_2_textfield'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Part of Body Two'),
+    '#default_value' => variable_get('drupal_form_api_part_of_body_2_textfield', ''),
+    //'#description' => t('Part of Body Two'),
```
Comment (fluxsauce) [inline drupal_form_api.module]: Remove commented out code.

File: drupal_form_api.module [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+    '#size' => 60,
+    '#maxlength' => 128,
+    '#required' => FALSE,
+  );
+  
+  $form['drupal_form_api_type_of_liquid_textfield'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Type of Liquid'),
+    '#default_value' => variable_get('drupal_form_api_type_of_liquid_textfield', ''),
+    //'#description' => t('Type of Liquid'),
```
Comment (fluxsauce) [inline drupal_form_api.module]: Remove commented out code.

File: drupal_form_api.module [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+    '#size' => 60,
+    '#maxlength' => 128,
+    '#required' => FALSE,
+  );
+  
+  $form['drupal_form_api_part_of_body_plural_textfield'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Part of Body Plural'),
+    '#default_value' => variable_get('drupal_form_api_part_of_body_plural_textfield', ''),
+    //'#description' => t('Part of Body Plural'),
```
Comment (fluxsauce) [inline drupal_form_api.module]: Remove commented out code.

File: drupal_form_api.module [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+      ),
+      'torso' => array(
+        0 => t('stomach'),
+      ),
+      'legs' => array(
+        0 => t('feet'),
+      ),
+    ),
+    '#default_value' => variable_get('drupal_form_api_part_of_body_3_select', ''),
+    //'#description' => t('Part of Body Three'),
```
Comment (fluxsauce) [inline drupal_form_api.module]: Remove commented out code.

File: drupal_form_api.module [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+    '#size' => 60,
+    '#maxlength' => 128,
+    '#required' => FALSE,
+  );
+  
+  $form['drupal_form_api_plural_noun_4_textfield'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Plural Noun Four'),
+    '#default_value' => variable_get('drupal_form_api_plural_noun_4_textfield', ''),
+    //'#description' => t('Plural Noun Four'),
```
Comment (fluxsauce) [inline drupal_form_api.module]: Remove commented out code.

File: drupal_form_api.module [hunk 8]
```diff
@@ -0,0 +1,234 @@
...
+    '#size' => 60,
+    '#maxlength' => 128,
+    '#required' => FALSE,
+  );
+    
+  $form['drupal_form_api_adjective_2_textfield'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Adjective Two'),
+    '#default_value' => variable_get('drupal_form_api_adjective_2_textfield', ''),
+    //'#description' => t('Adjective Two'),
```
Comment (fluxsauce) [inline drupal_form_api.module]: Remove commented out code.

File: drupal_form_api.module [hunk 9]
```diff
@@ -0,0 +1,234 @@
...
+    '#size' => 60,
+    '#maxlength' => 128,
+    '#required' => FALSE,
+  );
+  
+  $form['drupal_form_api_verb_ing_textfield'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Verb_ing'),
+    '#default_value' => variable_get('drupal_form_api_verb_ing_textfield', ''),
+    //'#description' => t('Verb_ing'),
```
Comment (fluxsauce) [inline drupal_form_api.module]: Remove commented out code.

File: drupal_form_api.module [hunk 10]
```diff
@@ -0,0 +1,234 @@
...
+    '#size' => 60,
+    '#maxlength' => 128,
+    '#required' => FALSE,
+  );
+  
+  $form['drupal_form_api_noun_1_textfield'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Noun 1'),
+    '#default_value' => variable_get('drupal_form_api_noun_1_textfield', ''),
+    //'#description' => t('Noun 1'),
```
Comment (fluxsauce) [inline drupal_form_api.module]: Remove commented out code.

File: drupal_form_api.module [hunk 11]
```diff
@@ -0,0 +1,234 @@
...
+    '#size' => 60,
+    '#maxlength' => 128,
+    '#required' => FALSE,
+  );
+    
+  $form['drupal_form_api_plural_noun_5_textfield'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Plural Noun Five'),
+    '#default_value' => variable_get('drupal_form_api_plural_noun_5_textfield', ''),
+    //'#description' => t('Plural Noun Five'),
```
Comment (fluxsauce) [inline drupal_form_api.module]: Remove commented out code.

File: drupal_form_api.module [hunk 12]
```diff
@@ -0,0 +1,234 @@
...
+    '#size' => 60,
+    '#maxlength' => 128,
+    '#required' => FALSE,
+  );
+  
+  $form['drupal_form_api_noun_2_textfield'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Noun Two'),
+    '#default_value' => variable_get('drupal_form_api_noun_2_textfield', ''),
+    //'#description' => t('Noun Two'),
```
Comment (fluxsauce) [inline drupal_form_api.module]: Remove commented out code.

**Comment by fluxsauce** [comment]
Looks good, :shipit: !

**Pull request merged**